package.json

```diff
@@ -48,49 +49,50 @@
           "singleQuote": true,
           "trailingComma": "es5",
           "bracketSpacing": false,
-          "jsxBracketSameLine": false
+          "jsxBracketSameLine": false,
+          "arrowParens": "always"
         }
       ]
     }
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-preset-shopify": "^16.1.0",
-    "eslint": "^4.7.1",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-shopify": "^16.2.0",
+    "eslint": "^4.14.0",
     "eslint-index": "^1.4.0",
     "eslint-plugin-shopify": "file:./.",
     "isparta": "^4.0.0",
     "istanbul": "^0.4.5",
-    "mocha": "^3.4.2",
-    "prettier": "^1.8.2",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "mocha": "^4.1.0",
+    "prettier": "^1.9.2",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0"
   },
   "peerDependencies": {
     "eslint": "<5 >=4.7.1"
   },
   "optionalDependencies": {
-    "prettier": "<2.0 >= 1.7.2"
+    "prettier": "<2.0 >= 1.9.2"
   },
   "dependencies": {
-    "babel-eslint": "^8.0.0",
-    "eslint-config-prettier": "^2.6.0",
-    "eslint-plugin-ava": "^4.2.1",
+    "babel-eslint": "^8.1.2",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-ava": "^4.4.0",
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-chai-expect": "^1.1.1",
-    "eslint-plugin-flowtype": "^2.32.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-lodash": "^2.4.4",
+    "eslint-plugin-flowtype": "^2.40.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-lodash": "^2.5.0",
```

Comment:
It looks like dependencies are just being bumped without updating our configs to reflect new rules / options.  Are we okay with just using whatever the defaults are?

e.g.,

`eslint-plugin-lodash@2.5.0` [introduces `import-scope`](https://github.com/wix/eslint-plugin-lodash/blob/master/CHANGELOG.md#added-1).  

`eslint-plugin-import@2.8.0` [introduces `exports-last`](https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md#added).

Reply:
Thanks for asking!

> eslint-plugin-lodash@2.5.0 introduces import-scope.

Looks like we already had a rule for that, maybe the yarn version was already set to 2.5.0.

```js
  // Prefer a specific import scope (e.g. lodash/map vs lodash)
  'lodash/import-scope': ['error', 'method'],
```

As for `exports-last`, it's disabled by default. Should I explicitly set it to `'off'`?

Reply:
I believe that we explicitly set a value for all options.  @lemonmade, that's correct?

Most likely, there are many more new rules + new options.  I'd rather someone trawl through changelogs and produce curated PRs, instead of a shotgun update.

Reply:
Yes, we try to provide explicit settings for all rules.

Reply:
> Most likely, there are many more new rules + new options. I'd rather someone trawl through changelogs and produce curated PRs, instead of a shotgun update.

I checked yarn.lock and it turns out we were already using the latest version of those 2 plugins. I guess the trick is to control which version is in the yarn.lock and make sure to document those upgrades too.

Let me go through the diffs and upgrade this PR.

Reply:
I've updated the body of the PR with a list of all new rules and a TODO. I'll add explicit settings for all those rules as well.

Reply:
@GoodForOneFare @lemonmade I've just updated the PR with new rules explicitly set to `off` when applicable. Can you please have another look?